core/src/test/java/org/apache/calcite/materialize/LatticeSuggesterTest.java

```diff
@@ -752,6 +756,93 @@ private void checkDerivedColumn(Lattice lattice, List<String> tables,
     checkDerivedColumn(lattice, tables, derivedColumns, 3, "n11", false);
   }
 
+  /** Test case for
+   * <a href="https://issues.apache.org/jira/browse/CALCITE-6605">[CALCITE-6605]
+   * Lattice SQL supports complex column expressions </a>. */
+  @Test void testExpressionLatticeSql() throws Exception {
```

Comment:
Maybe you can add some blank lines or a method to make the test sample code more readable.🤔

Reply:
Added description for tests.